src/components/orders/edit-order/EditOrder.js

```diff
@@ -0,0 +1,170 @@
+        if (newAdditionToOrder.productos.find(item => item.id === product.id)) {
+            const productos = newAdditionToOrder.productos
+                .map((item) => item.id === product.id ? { ...item, quantity } : item);
+            setNewAdditionToOrder({
+                ...newAdditionToOrder,
+                productos,
+                total: calculateTotal(productos)
+            });
+        } else {
+            const productos = [...newAdditionToOrder.productos, { ...product, quantity }];
+            setNewAdditionToOrder({
+                ...newAdditionToOrder,
+                productos,
+                total: calculateTotal(productos)
+            });
+        }
+    };
+
+    return (
+        <Fragment>
+            <div id="modal2" className="modal scroll mod-edit-options">
+                <div className='info-table-selected'>
+                    <p className='table-num'>Mesa  {orderSelected.table}</p>
+                </div>
+                <div className="modal-content mod-content-edit">
+                    <div className='add-products-order'>
+                        <table className='table-new-items-added'>
+                            <thead>
+                                <tr>
+                                    <th>Productos ordenados</th>
+                                    <th>Cantidad</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {orderSelected.order.map((item) => (
+                                    <tr>
+                                        <td>
+                                            <div className='product-added'>
+                                                <button type='button' className='btn-delete-item-added'>X</button>
+                                                <p className='item-text-table'>{item.item}</p>
+                                            </div>
+                                        </td>
+                                        <td>
+                                            <p>{item.quantity} </p>
+                                        </td>
+                                    </tr>
+                                ))}
+                            </tbody>
+                        </table>
+                    </div>
+                    <div className='add-products-order'>
+                        <table className='table-new-items-added'>
+                            <thead>
+                                <tr>
+                                    <th>Productos Agregados</th>
+                                    <th>Cant.</th>
+                                    <th>Precio total</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {
+                                    newProduct.map((item) => (
```

Comment:
Cada que haces un .map()  tienes que usar key en los elementos renderizáados
```suggestion
                                    newProduct.map((item) => (
```